netjson-monitoring.lua

```diff
@@ -320,12 +415,40 @@ for name, interface in pairs(network_status) do
   if not is_excluded(name) and not processed[name] then
     netjson_interface = {
         name = name,
-        statistics = interface.statistics
+        type = string.lower(interface.type),
+        up = interface.up,
+        mac = interface.macaddr,
+        txqueuelen = interface.txqueuelen,
+        mtu = interface.mtu,
+        speed = interface.speed,  -- XXX: untested
+        bridge_members = interface['bridge-members'],
+        stp = interface.stp,  -- XXX: untested
     }
+    if interface.type == 'Network device' then
+        link_supported = interface['link-supported']
+        if link_supported and next(link_supported) then
+            -- XXX: untested
```

Comment:
we can remove this, I tested it on my device and it works, we'll keep improving it upon further testing in the future